srcs/exec/handle_node.c

```diff
@@ -22,21 +22,23 @@ t_result	exec_handle_left_node(t_ast *self_node, t_context *context)
 }
 
 // &&, ||
-static bool	is_executable_right_node(t_ast *self_node, uint8_t status)
+static bool	is_executable_right_node(t_ast *self_node, t_context *context)
 {
+	if (context->is_return)
+		return (false);
 	if (!is_node_kind_and_or(self_node->kind))
 		return (true);
-	if (self_node->kind == NODE_KIND_OP_AND && status == 0)
+	if (self_node->kind == NODE_KIND_OP_AND && context->status == 0)
 		return (true);
-	if (self_node->kind == NODE_KIND_OP_OR && status != 0)
+	if (self_node->kind == NODE_KIND_OP_OR && context->status != 0)
```

Comment:
マクロ使いますか。31行目も。 (消した行？は suggestion できなかった…)
```suggestion
	if (self_node->kind == NODE_KIND_OP_OR && context->status != EXIT_SUCCESS)
```